services/motion/builtin/move_request.go

```diff
@@ -134,6 +135,8 @@ func kbOptionsFromCfg(motionCfg validatedMotionConfiguration, validatedExtra val
 	}
 
 	kinematicsOptions.GoalRadiusMM = motionCfg.planDeviationMM
+	// NOTE: Are we sure that being tolerant of being off the goal by 8 degress isn't
```

Comment:
So far this hasn't been implicated in anything; personally if I ask for the robot to point West and it winds up within 8 degrees of that I'd have trouble telling by eyeballing it.

Reply:
that makes sense, I would too